Corona-Warn-App/src/main/res/values/styles.xml

```diff
@@ -104,6 +104,11 @@
         <item name="android:textColor">@color/button_text_color_emphasized</item>
     </style>
 
+    <style name="debugLogButton" parent="button">
```

Comment:
Same here, `debugLogButton` is very generic. Generic is okay if we want to re-use it in the app, but then maybe name it  something like `buttonBarAlertRed` or so.